### ISSUE TITLE
ImportError: cannot import name 'img_as_float32'

### ISSUE THREAD

**Issue by mingghan** [retentioneering/retentioneering-tools#7]
Hi guys

I tried to get through your "first steps" guide but it raised an error on the start

```
from retentioneering import init_config
import pandas as pd
```

```
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-21-fe4dd999c90a> in <module>()
----> 1 from retentioneering import init_config
      2 import pandas as pd

/anaconda3/lib/python3.6/site-packages/retentioneering/__init__.py in <module>()
----> 1 from retentioneering.core.utils import init_config
      2 from retentioneering.core.simulator import Simulator

/anaconda3/lib/python3.6/site-packages/retentioneering/core/utils.py in <module>()
     14 from retentioneering.visualization import plot
     15 from sklearn.linear_model import LogisticRegression
---> 16 from retentioneering.core.model import ModelDescriptor
     17 from retentioneering.core import node_metrics
     18 from retentioneering.core import preprocessing

/anaconda3/lib/python3.6/site-packages/retentioneering/core/model.py in <module>()
      5 
      6 
----> 7 import shap
      8 import eli5
      9 from eli5.sklearn import PermutationImportance

/anaconda3/lib/python3.6/site-packages/shap/__init__.py in <module>()
     10 from .explainers.linear import LinearExplainer
     11 from .explainers.partition import PartitionExplainer
---> 12 from .plots.summary import summary_plot
     13 from .plots.decision import decision_plot, multioutput_decision_plot
     14 from .plots.dependence import dependence_plot

/anaconda3/lib/python3.6/site-packages/shap/plots/summary.py in <module>()
     13     pass
     14 from . import labels
---> 15 from . import colors
     16 
     17 # TODO: remove unused title argument / use title argument

/anaconda3/lib/python3.6/site-packages/shap/plots/colors.py in <module>()
      5 
      6 import numpy as np
----> 7 import skimage.color
      8 
      9 try:

/anaconda3/lib/python3.6/site-packages/skimage/color/__init__.py in <module>()
----> 1 from .colorconv import (convert_colorspace,
      2                         guess_spatial_dimensions,
      3                         rgba2rgb,
      4                         rgb2hsv,
      5                         hsv2rgb,

/anaconda3/lib/python3.6/site-packages/skimage/color/colorconv.py in <module>()
     54 import numpy as np
     55 from scipy import linalg
---> 56 from ..util import dtype, dtype_limits
     57 
     58 

/anaconda3/lib/python3.6/site-packages/skimage/util/__init__.py in <module>()
----> 1 from .dtype import (img_as_float32, img_as_float64, img_as_float,
      2                     img_as_int, img_as_uint, img_as_ubyte,
      3                     img_as_bool, dtype_limits)
      4 from .shape import view_as_blocks, view_as_windows
      5 from .noise import random_noise

ImportError: cannot import name 'img_as_float32'
```

skimage version: 0.15.0

**Comment by tokedo** [comment]
Hello, 

This is the problem with the shap library. You can try to update the shap library on your system. This might fix the problem. 

Actually, this library is rarely used and we removed it from the list of main requirements at the latest update (v1.0.4). Please try again later today, we are planning to roll out an update soon.

**Comment by mingghan** [comment]
Thank you for the quick response. I tried to upgrade, but it didn't help

> !pip3 install --upgrade shap
> Requirement` already up-to-date: shap in /anaconda3/lib/python3.6/site-packages (0.30.1)

**Comment by tokedo** [comment]
try to update the Retentioneering library to the latest version 1.0.4, we've just released the update.

**Comment by mingghan** [comment]
It worked now, thank you!

**Comment by tokedo** [comment]
perfect! let us know if you have any other issues
